deploy-manage/distributed-architecture/shard-allocation-relocation-recovery/shard-allocation-awareness.md

```diff
@@ -1,6 +1,8 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/shard-allocation-awareness.html
+applies_to:
+  stack:
```

Comment:
This page (e.g. https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/561/deploy-manage/distributed-architecture/shard-allocation-relocation-recovery/shard-allocation-awareness) is another one where I see specific reference to editing `elasticsearch.yml` so either the `applies_to` needs to have the `self` deployment type added or else the step about how to edit elasticsearch settings needs to be generalized

Reply:
setting as self until we go and do a full review.